src/components/FindMovie/FindMovie.js

```diff
@@ -1,55 +1,112 @@
+
 import './FindMovie.scss';
 
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
 import { MovieCard } from '../MovieCard';
-import movies from '../../api/movies.json';
-
-export const FindMovie = () => (
-  <>
-    <form className="find-movie">
-      <div className="field">
-        <label className="label" htmlFor="movie-title">
-          Movie title
-        </label>
-
-        <div className="control">
-          <input
-            type="text"
-            id="movie-title"
-            placeholder="Enter a title to search"
-            className="input is-danger"
-          />
-        </div>
+import { getMovie } from '../../api/api';
 
-        <p className="help is-danger">
-          Can&apos;t find a movie with such a title
-        </p>
-      </div>
+export const FindMovie = ({ addMovie }) => {
+  const [movie, setMovie] = useState(null);
+  const [title, setTitle] = useState('');
+  const [error, setError] = useState(false);
+
+  const findMovie = () => {
+    getMovie(title)
+      .then((data) => {
+        if (data.Response === 'False') {
+          setError(true);
+          setMovie(null);
+
+          return;
+        }
 
-      <div className="field is-grouped">
-        <div className="control">
-          <button
-            type="button"
-            className="button is-light"
+        setMovie({
+          title: data.Title,
+          description: data.Plot,
+          imgUrl: data.Poster,
+          imdbUrl: `https://www.imdb.com/title/${data.imdbID}/`,
+          imdbId: data.imdbID,
+        });
+        setTitle('');
+      });
+  };
+
+  return (
+    <>
+      <form
+        className="find-movie"
+      >
+        <div className="field">
+          <label
+            className="label"
+            htmlFor="movie-title"
           >
-            Find a movie
-          </button>
+            Movie title
+          </label>
+
+          <div className="control">
+            <input
+              type="text"
+              id="movie-title"
+              placeholder="Enter a title to search"
+              className={classNames(
+                'input', { 'is-danger': error },
+              )}
+              value={title}
+              onChange={(event) => {
+                setTitle(event.target.value);
+                setError(false);
+              }}
+            />
+          </div>
+          {error && (
+            <p className="help is-danger">
+              Can&apos;t find a movie with such a title
+            </p>
+          )}
         </div>
 
-        <div className="control">
-          <button
-            type="button"
-            className="button is-primary"
-          >
-            Add to the list
-          </button>
+        <div className="field is-grouped">
+          <div className="control">
+            <button
+              type="button"
+              className="button is-light"
+              onClick={findMovie}
+            >
```

Comment:
let me and other users press 'enter' and don't refresh the page, also it's cool if the user can just type the name of the movie and search for it with the `enter` button.